pkg/sql/alter_table_locality.go

```diff
@@ -326,3 +268,75 @@ func (n *alterTableSetLocalityNode) startExec(params runParams) error {
 			TableName: n.n.Name.String(),
 		})
 }
+
+func generateNewLocalityConfigForLocalityGlobal() *descpb.TableDescriptor_LocalityConfig {
+	return &descpb.TableDescriptor_LocalityConfig{
+		Locality: &descpb.TableDescriptor_LocalityConfig_Global_{
+			Global: &descpb.TableDescriptor_LocalityConfig_Global{},
+		},
+	}
+}
+
+func generateNewLocalityConfigForLocalityRegionalByTable(
```

Comment:
nit: i'd get rid of `generate` or `new` -- `generateLocalityConfigFor...` or `newLOcalityFor...`